examples/hello_world.js

```diff
@@ -69,14 +69,13 @@ function main() {
   }
 
   // Delete objects we created through |new|. We just do a few of them here, but you should do them all if you are not shutting down ammo.js
-  Ammo.destroy(collisionConfiguration);
-  Ammo.destroy(dispatcher);
-  Ammo.destroy(overlappingPairCache);
+  Ammo.destroy(dynamicsWorld);
```

Comment:
was the issue that the order needs to be like this? if so then perhaps add a comment here?